src/vendor/wordpress/_wordpress.scss

```diff
@@ -111,6 +106,47 @@ $wp-button-gap: sizes.$list-inline-gap;
   }
 }
 
+/**
+ * Gutenberg block: Image
+ * Styles for images inserted via Gutenberg blocks.
+ *
+ * 1. Since we are applying `margin-top` to CMS content via our staggerItems
+ *    utility, the default `margin-bottom` value on Gutenberg image blocks is
+ *    unnecessary, so we remove it here.
+ */
+.wp-block-image {
+  margin-bottom: 0; /* 1 */
+  position: relative;
+
+  /**
+   * 1. Setting a width of `100%` prevents resized and centered images from
+   *    breaking layout when placed after a right-aligned image in a post.
+   */
+  &.aligncenter {
+    width: 100%; /* 1 */
+  }
+
+  &.alignfull {
```

Comment:
Consider adding a preceding comment linking to the article this was pulled from for context.

Reply:
The GitHub UI isn't showing this comment as "outdated" but it was updated in a190909